includes/validation/class-amp-validation-callback-wrapper.php

```diff
@@ -393,6 +393,7 @@ protected function finalize_scripts( WP_Scripts $wp_scripts, array $before_regis
 	 * @param mixed $offset Offset.
 	 * @param mixed $value  Value.
 	 */
+	#[\ReturnTypeWillChange]
```

Comment:
I suppose this is needed because we can't specify the matching return type since return types aren't available in older versions of PHP. 

Oh, but actually we now require PHP 7 in the AMP plugin. So instead of this attribute could we instead just add the return types?

Reply:
If we‘re not worried about someone extending the class and breaking their implementation because of the return types, sure.

Reply:
See https://php.watch/versions/8.1/ReturnTypeWillChange

Reply:
No, I'm not worried about someone extending the class. It should have been marked `final` to begin with.

Reply:
Since we can't use `void` return types yet (they're a PHP 7.1 feature), can't really change this.